Source/ASButtonNode.mm

```diff
@@ -562,4 +562,11 @@ - (void)layout
   _titleNode.hidden = (_titleNode.attributedText.length == 0);
 }
 
+- (void)tintColorDidChange
+{
+  [super tintColorDidChange];
+  _imageNode.imageModificationBlock = ASImageNodeTintColorModificationBlock(self.tintColor);
```

Comment:
This will override any modification block set by clients beforehand. 

You may want to implement a new block that first calls the existing block, if any, and then calls a tint color block. Even this won't prevent users from setting their own block after changing the tint color and thus not seeing the color applied. We may want something like a "final tint color modification block" that is guaranteed to be executed last. Perhaps by adding an ASImageNode subclass?

Reply:
Yes, just want to use only imageNode of ASButtonNode.
So, How about null check about imageModificationBlock of ASButtonNode imageNode property?

> Avoid any modification block set by clients beforehand.
```objc
- (void)tintColorDidChange
{
  [super tintColorDidChange];
  if !(_imageNode.imageModificationBlock) {
    _imageNode.imageModificationBlock = ASImageNodeTintColorModificationBlock(self.tintColor);
    [_imageNode setNeedsDisplay];
  }
}
```
Also, if this method doesn't good then i will close this PR :) 

Thank you!